### ISSUE TITLE
Feature Request: Disable Distance-Based Scaling

### ISSUE THREAD

**Issue by someONEgood** [fitzchivalrik/compass#10]
TL;DR: Basically I'd like to be able to have FATE icons at 100% size regardless of distance.

Would it be possible to add the ability to toggle/disable if/how icons (specifically FATES) scale based on distance?
I'm using your plugin almost exclusively for FATE navigation be it Bozja/Zadnor or open world zones, and it would be great if pairing it with Map as Source allowed all FATEs in the zone to be showed on the compass.

Currently that's not entirely possible (unless I'm missing a setting) since FATES at a certain distance don't appear because they are scaled super tiny when far away. Absolutely love this plugin and how sleek it is. Thanks!

**Comment by fitzchivalrik** [comment]
Thanks for your kind works!

Just to be sure, I was talking about this distance setting:
![image](https://user-images.githubusercontent.com/77893938/136466594-7a576cec-c4bb-45d8-9a8f-eb89991f1caa.png)

Regardless, I will look into exposing the minimum scaling as a setting.

**Comment by fitzchivalrik** [comment]
All of them are used for different sized circles afaiu.

After EW I wanted to have another look at how Compass works in the backend and see if maybe some new discoveries of the incredible people at Dalamud and my greater knowledge will allow me to rewrite Compass to something better - and maybe fixing problems like that at the same time.

I won't take another look before and just after EW release though, maybe around December.
